lib/src/components/NxCodeSnippet/NxCodeSnippet.scss

```diff
@@ -42,6 +42,10 @@
   .nx-text-input {
     grid-area: textarea;
     width: 100%;
+    // When the rows attribute is specified for a textarea, Firefox adds an additional empty row at 
+    // the bottom of the textarea. Overflow-x:hidden resolves and removes the empty row
+    // https://stackoverflow.com/questions/46746887/textarea-extra-row-in-firefox-browser
+    overflow-x: hidden;
```

Comment:
Oh I think this is actually on the wrong element. Doesn't it need to be on the `textarea`?  This likely explains the visual test issue.

Reply:
yes, that was the intention! Thanks for catching it, updated here: [7a895b0](https://github.com/sonatype/sonatype-react-shared-components/pull/743/commits/7a895b056b3c60ff372989d373235a1361f0c067)

Visual checks now passing